stdlib/stdlib.mli

```diff
@@ -41,6 +41,13 @@ val invalid_arg : string -> 'a
 val failwith : string -> 'a
 (** Raise exception [Failure] with the given string. *)
 
+val todo : ?msg:string -> ?__LOC__:string -> unit -> _
```

Comment:
I don’t know what I think about labelling an argument `__LOC__`. In my mind this name is reserved as a lexer placeholder, and the use of the function looks verbose to em. To get the location of the error, we have exception backtraces.

Reply:
Fair enough, but i was going from the convention of passing explicit __LOC__ from `tezt` or something akin to `ppx_here`.
But yeah, given the fact that user can just call `Printexc.record_backtrace true` this seems excessive. removed

Reply:
Using `__LOC__` is a hack that I think I've seen in other libraries. There's no backtrace if compiled without debug information. This could be solved elegantly with [Automatically insert source location #126](https://github.com/ocaml/ocaml/pull/126).